thirdparty/boost/CMakeLists.txt

```diff
@@ -16,13 +15,7 @@ set(Boost_B2_FLAGS variant=release threading=multi runtime-link=shared link=shar
 # Note: Since all builds below depend on the headers, use this step to
 # download the boost release
 
-ExternalProject_Add(
-  boost
-  URL ${Boost_URL}
-  CONFIGURE_COMMAND ""
-  BUILD_COMMAND ""
-  INSTALL_COMMAND ""
-)
+ExternalProject_Add( boost "${ACTS_BOOST_SOURCE};CONFIGURE_COMMAND;;BUILD_COMMAND;;INSTALL_COMMAND;" )
```

Comment:
that command looks a bit obscure - is this a functional change?

Reply:
Oh, I missed that - @fred can you comment?

Reply:
I changed it so that the source is list (which can contain URL, MD5, etc) and also to make it consistent with how the rest of the external projects are fetched.

Reply:
But don't be worried to mix the two formalisms. The following should work fine:

```suggestion
ExternalProject_Add(
   boost  
   ${ACTS_BOOST_SOURCE}
   CONFIGURE_COMMAND "" 
   BUILD_COMMAND ""
   INSTALL_COMMAND "" )
```

CMake will understand what you meant by that. This is how I do things in the offline code as well.
  - https://gitlab.cern.ch/atlas/atlasexternals/-/blob/main/External/CLHEP/CMakeLists.txt?ref_type=heads#L61-71
  - https://gitlab.cern.ch/atlas/atlasexternals/-/blob/main/External/Acts/CMakeLists.txt?ref_type=heads#L58-74
  - etc.

Reply:
To me that version is more comprehensible. Maybe something we can follow up on